src/transforms/transform-commit.ts

```diff
@@ -16,7 +17,7 @@ export const mapCommit = (commit): JiraCommit | undefined => {
 		hash: commit.oid,
 		id: commit.oid,
 		issueKeys,
-		message: commit.message,
+		message: commit?.message?.substring(0, MAX_COMMIT_MESSAGE_LENGTH),
```

Comment:
hm, I'm starting to thing we need to create a function that does this as a util so we don't have to replicate logic.

Reply:
Sweet, I will move it all to one location.